store/sqlstore/retention_policy_store.go

```diff
@@ -811,11 +999,11 @@ func genericRetentionPoliciesDeletion(
 		` + query + `
 		) AS A ON ` + joinClause
 	}
-	result, err := s.GetMaster().Exec(query, args...)
+	sqlResult, err := s.GetMasterX().Exec(query, args...)
```

Comment:
Nit: It's okay to keep the variable name as `result`. Feel free to do it in other place also.